source/tutorials/sample-classifier.rst

```diff
@@ -83,6 +83,13 @@ In addition to the predicted class information, the model also reports the indiv
      --m-input-file moving-pictures-classifier/probabilities.qza \
      --o-visualization moving-pictures-classifier/probabilities.qzv
 
+Besides outputting the predictions and probabilities of the test samples, the pipeline also returns the true target values of all train and test samples in ``training_targets.qza`` and ``test_targets.qza`` respectively. Both are ``SampleData[TrueTargets]`` artifacts that can be viewed as metadata and inspected with ``metadata tabulate``, e.g. for ``test_targets.qza``:
```

Comment:
Minor change: I recommend breaking up the last sentence for clarity:
", e.g. for ``test_targets.qza``:" --> ". For example, view true and predicted values for the test samples:"